### PR TITLE
feat(ab framework): update api url

### PR DESCRIPTION
**Pull request by danilowoz**

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5920/commits/3ce41dc86647b022ce1d97d19efbeb21cbdcfb9b is at https://pr5920.build.csb.dev/s/new.

**Review by CompuIves — APPROVED** [review]
Great! Merci!

File: packages/common/src/config/env.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ const NODE_ENV = JSON.stringify(process.env.NODE_ENV || 'development');
 const LOCAL_SERVER = Boolean(JSON.stringify(process.env.LOCAL_SERVER));
 const STAGING_API = Boolean(JSON.stringify(process.env.STAGING_API));
 
-export const AB_TESTING_URL = 'https://ab-testing.codesandbox.workers.dev';
+export const AB_TESTING_URL = 'https://ab-testing.codesandbox.io';
```
Comment (danilowoz) [inline packages/common/src/config/env.ts:11]: It should be https://ab-testing.codesandbox.stream/ on staging and https://ab-testing.codesandbox.io/ in production.

File: packages/common/src/config/env.ts [hunk 2]
```diff
@@ -8,7 +8,11 @@ const NODE_ENV = JSON.stringify(process.env.NODE_ENV || 'development');
 const LOCAL_SERVER = Boolean(JSON.stringify(process.env.LOCAL_SERVER));
 const STAGING_API = Boolean(JSON.stringify(process.env.STAGING_API));
 
-export const AB_TESTING_URL = 'https://ab-testing.codesandbox.workers.dev';
+const AB_TESTING_URL_STAGING = 'https://ab-testing.codesandbox.stream';
+const AB_TESTING_URL_PRODUCTION = 'https://ab-testing.codesandbox.io';
+export const AB_TESTING_URL = STAGING_API
```
Comment (danilowoz) [inline packages/common/src/config/env.ts:13]: @CompuIves Does it make sense? I afraid it doesn't work, as @lbogdan pointed out, we use the same build for staging and production, so the choice needs to be done at runtime.

I grabbed this logic from [here](https://github.com/codesandbox/codesandbox-client/blob/master/packages/common/src/components/Preview/index.tsx#L76), so hope it makes sense.

Comment (CompuIves) [reply packages/common/src/config/env.ts:13]: Oh yeah we could even just make it io at all times. That seems fine with me as well.

**Review by CompuIves — APPROVED** [review]

**Pull request merged**